ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ImageNameTd.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
-import { ClipboardCopyButton, Flex, FlexItem, Truncate } from '@patternfly/react-core';
```

Comment:
@mansursyed 
I noticed that Dave had to remove using [an existing PatternFly component](https://v4-archive.patternfly.org/v4/components/clipboard-copy), the **CliplboardCopyButton**, and build his own custom button, to fulfill the UX request for this change.

I think if we're going to make this sort of change, we should advocate that the PatternFly team adopt this new style in their component. Having ACS-only variants is not efficient or sustainable.